sentinelhub/migration_from_scihub_guide.ipynb

```diff
@@ -928,9 +948,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Sentinel Hub",
+   "display_name": "Python 3 (ipykernel)",
    "language": "python",
-   "name": "sentinelhub"
+   "name": "python3"
```

Comment:
Please correct the kernelspec metadata according to the [instructions in the repo](https://github.com/eu-cdse/notebook-samples#kernel-spec-metadata).

Reply:
resolved.